builtin/v12/migration/miner.go

```diff
@@ -120,7 +149,7 @@ func (m minerMigrator) MigrateState(ctx context.Context, store cbor.IpldStore, i
 
 	wrappedStore := adt11.WrapStore(ctx, store)
 
-	newSectors, err := migrateSectorsWithCache(ctx, wrappedStore, in.Cache, in.Address, inState.Sectors)
+	newSectors, err := m.migrateSectorsWithCache(ctx, wrappedStore, in.Cache, in.Address, inState.Sectors)
```

Comment:
(not related to this PR) 

the portion above this should be dropped entirely -- we are not migrating post proof types / minerinfos in this network upgrade.

Reply:
@snissn Can you make this change too (ideally as a pre-factor that we land first, but otherwise just as part of this PR)?

Reply:
I'm worried about forgetting about it.